Dockerfile

```diff
@@ -0,0 +1,6 @@
+# Metaflow bot docker file : Temp workaround like to to use heroku github action
```

Comment:
Can we move this under `metaflow-bot/` directory

Reply:
I kept it here because of Heroku deployments; Was a quick hacky decision; It can be moved into the bot code.